lib/charms/observability_libs/v1/kubernetes_service_patch.py

```diff
@@ -0,0 +1,341 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
I don't think this gets used in this PR, does it?  It looks like it is being added, but charm.py is not changing

Reply:
Prometheus lib uses the 

```
from charms.observability_libs.v0.juju_topology import JujuTopology
```